server/src/services/s3.js

```diff
@@ -0,0 +1,54 @@
+import AWS from "aws-sdk";
+
+export const uploadFileToS3 = async (path, file, fileName, contentType) => {
+  try {
+    if (process.env.REACT_APP_ENVIRONMENT === "development") {
```

Comment:
Any reasons why we're including React env in server-side code? I think the convention is to use `process.env.NODE_ENV`, which should be set to dev mode by Express?

Reply:
Oh bữa em set cái này mà em ko để ý naming. Có gì @lnguyen2693 đổi lại thành NODE_ENV chỗ này + update env docs + sửa lại code